src/pages/help-centre/index.tsx

```diff
@@ -9,11 +9,15 @@ import { SEO, Desktop, Container } from 'components/containers'
 import Layout from 'components/layout/layout'
 import { localize, WithIntl } from 'components/localization'
 import { usePlatformQueryParam } from 'components/hooks/use-platform-query-param'
+import { DerivStore } from 'store'
 
 const HelpCentre = () => {
     const { is_deriv_go } = usePlatformQueryParam()
+    const { is_p2p_allowed_country } = React.useContext(DerivStore)
     const general_questions = getQuestionsBySection(GENERAL)
-    const platforms_questions = getQuestionsBySection(PLATFORMS)
+    const platforms_questions = getQuestionsBySection(PLATFORMS).filter((item) => {
+        return !is_p2p_allowed_country ? !item.category.includes('_t_Deriv P2P_t_') : item
```

Comment:
Hi @aswathy-deriv , in the include, do we need to have the "_t_Deriv P2P_t_" translation key?

Reply:
okay i removed it